src/cc_helpers.cc

```diff
@@ -412,6 +442,11 @@ void certifier::framework::cc_trust_data::print_trust_data() {
   } else {
     printf("all not initialized\n");
   }
+
+  printf("Number of certifiers: %d\n", num_certified_domains_);
+  for (int i = 0; i < num_certified_domains_; i++) {
+    certified_domains_[i]->print_certifiers_entry();
```

Comment:
I still don't understand domain add / drop business.

But if in `certify_secondary_domain()`, for example, we are checking for null entry while walking the array, should we not apply the same coding style here, too? And only print if [i]'th entry is non-Null?